src/modules/public/alert/sagas/handleAlertSaga.ts

```diff
@@ -41,6 +40,14 @@ export function* handleAlertSaga(action: AlertPush) {
                 }
 
                 return;
+            case 471:
+                localStorage.setItem('restricted', 'true');
+                window.location.replace('/404');
+                break;
+            case 472:
+                localStorage.setItem('maintenance', 'true');
+                window.location.replace('/500');
+                break;
```

Comment:
🚨!!! Please, don't use browser storage (localStorage) here !!!🚨
Once it set 404, then every page refresh, doesn't matter if server is ok, it redirects user to restriction / maintenance page.